### ISSUE TITLE
problem with "Multiple Projects" feature

### ISSUE THREAD

**Issue by shuji-koike** [AtomLinter/linter-eslint#1191]
## Issue Type
Bug?

## Issue Description
`linter-eslint` is not working properly when "Multiple Projects" are opened in a single window.
No errors are reported and Fix on Save feature is not also working.

## Bug Checklist
- [x] Restart Atom
- [x] Verify the `eslint` CLI gives the proper result, while `linter-eslint` does not
- [ ] Paste the output of the `Linter Eslint: Debug` command from the Command Palette below

**Comment by shuji-koike** [comment]
> Paste the output of the Linter Eslint: Debug command from the Command Palette below

"Linter Eslint: Debug" does not showup in the pallet.
BTW I have `linter-eslint@8.4.1` installed.

**Comment by IanVS** [comment]
Hi @shuji-koike, have you restarted atom lately?  

And can you describe what you mean by multiple projects open in a single window?  I'm not certain exactly what you mean.  Thanks.

**Comment by shuji-koike** [comment]
Hi @IanVS 
I have restarted now and before I posted this issue. But still not working 😿 

> multiple projects open in a single window

I meant when using this feature, which is in atom out of the box.
http://blog.atom.io/2015/04/15/multi-folder-projects.html

`linter-eslint` is working fine in most of my windows except for the one that has multi folder projects.

**Comment by skylize** [comment]
> linter-eslint is not working properly when "Multiple Projects" are opened in a single window.

So this is only a problem with multiple projects open? What do you get from `Linter Eslint: Debug ` with only one project open?

Are there any exceptions showing up in Dev Tools console?

**Comment by shuji-koike** [comment]
> So this is only a problem with multiple projects open?

yes.
Nothing is logged in Dev Tools console.

here is my out put take from a **working** window
```
Atom version: 1.32.0
linter-eslint version: 8.4.1
ESLint version: 4.19.1
Hours since last Atom restart: 0
Platform: darwin
Using local project ESLint from: *****
Current file's scopes: [
  "source.js"
]
linter-eslint configuration: {
  "fixOnSave": true,
  "lintHtmlFiles": true,
  "rulesToDisableWhileFixing": [
    "no-debugger"
  ],
  "scopes": [
    "source.js",
    "source.jsx",
    "source.js.jsx",
    "source.babel",
    "source.js-semantic",
    "text.html.vue"
  ],
  "useGlobalEslint": false,
  "showRuleIdInMessage": true,
  "disableWhenNoEslintConfig": true,
  "eslintrcPath": "",
  "globalNodePath": "",
  "advancedLocalNodeModules": "",
  "eslintRulesDirs": [],
  "disableEslintIgnore": false,
  "disableFSCache": false,
  "rulesToSilenceWhileTyping": [],
  "ignoreFixableRulesWhileTyping": false
}
```

Somehow, `Linter Eslint: Debug` command is missing in the **broken** window, so unable to call it. 🤔

**Comment by skylize** [comment]
Cannot reproduce this.

Is it happening with every project, or just some? Are *both* projects working fine alone?
